src/main/java/racingcar/ui/InputView.java

```diff
@@ -0,0 +1,17 @@
+package racingcar.ui;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private final static Scanner SCANNER = new Scanner(System.in);
```

Comment:
상수 네이밍 컨벤션에 따라 `static final` 순으로 작성해주시면 좋을 것 같습니다.
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```